recipes/ansible-base/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ansible-base" %}
+{% set version = "2.10.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 72c062ccdcec627542c8305ca7050cbdc651585a3fa27e3f536f82a27b5e0557
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4
```

Comment:
Do you need this on Python 2, or can this be `>=3.5`? Same below.

Reply:
technically does still support 2.7